### DISCUSSION TITLE
feature: ability to set amount range for items drop within the NPC editor

### DISCUSSION THREAD

**Discussion by Superspyi** [AscensionGameDev/Intersect-Engine#2209] [Ideas]
**Is your feature request related to a problem? Please describe.**
Yes. What I would like to do is make an NPC drop a random amount of an item within a certain range. For Example:

- For early game NPCs I would like for them to drop between 1-5 gold as one item to pickup.
- End game bosses would be much larger between 1200-2400 gold.

Currently I when I create an NPC with drops such as gold I have to set a static amount of gold to drop at a certain percentage. The only way I can get close to what I'd like is to create multiple drops of the same item of 1x at the same percentage but then the user would have to pick up multiple items. This becomes impossible for something like end game bosses.

**Describe the solution you'd like**
Introduce a "range" option near the amount section in NPC Editor.

**Additional context**
I believe this would be a great feature to add into the Intersect engine due to most RPGs I've played before having random amounts of currency or items on NPCs when defeated such as $ in RDR2 or Gold in Oblivion & Skyrim. Attached is statistics for gold from an NPC on another MMORPG that I play that includes amount ranges.
![AmountRange](https://user-images.githubusercontent.com/64320948/82597566-27d67100-9b77-11ea-91a7-0f75e925230d.PNG)

**Comment by Arufonsu** [comment]
Added to [_(v0.8.0-beta.120+) Roadmap_](https://github.com/AscensionGameDev/Intersect-Engine/discussions/2303)

**Comment by WeylonSantana** [comment]
done in https://github.com/AscensionGameDev/Intersect-Engine/pull/2486
